modules/custom/cu_seo_bundle/cu_seo_bundle.info

```diff
@@ -5,6 +5,8 @@ core = 7.x
 package = CU-Search Engine Optimization
 project_demo_url = http://www.colorado.edu/webcentral/node/684
 dependencies[] = metatag
+dependencies[] = metatag_opengraph
+dependencies[] = metatag_twitter_cards
```

Comment:
I don't see the description tag. I tried adding text to the summary part of the node form as well as manually setting the text to "and stuff..."